.github/workflows/ci.yaml

```diff
@@ -60,6 +60,14 @@ jobs:
       - name: Test
         run: cargo test -- --nocapture --quiet
 
+      - name: Build fuzzers
```

Comment:
I'm not sure we want to do this here, we try to keep the CI times relatively low.

Reply:
Is there somewhere else that you'd want to put this? Or would you prefer not to have this captured in the CI at all?

Reply:
The intention here is just to build these fuzz harnesses to prevent future bitrot. These harnesses aren't being run, as that certainly would tank the CI times.

Reply:
oops, sorry for some reason I didn't see your [second message](https://github.com/rust-lang/rust-analyzer/pull/14010#discussion_r1084432975)... I could add it as a separate workflow, with a [cron trigger](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onschedule) once a week or so?

Reply:
Yeah, I mean even if it's a couple of minutes to install `cargo-fuzz` and build the harness (`libfuzzer-sys` seems quite large), that's more than we want given how running the fuzzer isn't very useful.

Reply:
> oops, sorry for some reason I didn't see your https://github.com/rust-lang/rust-analyzer/pull/14010#discussion_r1084432975... I could add it as a separate workflow, with a [cron trigger](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onschedule) once a week or so?

That sounds completely reasonable.

Reply:
Done here https://github.com/rust-lang/rust-analyzer/pull/14010/files#diff-05173c6260b3a634672ba19a936ae023b6635bf308b195d5eaf9b986021184d4R5